docs/pfsense_installerconfig.md

```diff
@@ -1,11 +1,62 @@
-Unattanded installation PFSense
+Unattended installation PFSense
 ===============================
 
+This document describes the process of preparing the automatic installer pFsense
+on a USB stick. The whole procedure was tested on Ubuntu with the kernel
+5.15.0-58-generic, the download and installation procedure of the kernel is also
+described below.
+
+Kernel installation
+-------------------
+
+The following set of commands should provide the successful installation of the
+kernel in the 5.15.0-58-generic version:
+
+```bash
+sudo apt install linux-headers-5.15.0-58-generic
+sudo apt install linux-source-5.15.0
+cd /usr/src/
+cd linux-source-5.15.0/
+sudo tar xjf  linux-source-5.15.0.tar.bz2 
+sudo cp ../linux-headers-5.15.0-58-generic/Module.symvers linux-source-5.15.0/
+cd linux-source-5.15.0/
+sudo cp /boot/config-5.15.0-58-generic .config
+sudo make olddefconfig
+```
+
+UFS module compilation
+-----------------------
+
+1. Edit the `.config` file by setting the value of `UFS_FS_WRITE` to `y`.
+
+1. The following set of commands provide for UFS module compilation:
+
+   ```bash
+   sudo make prepare
```

Comment:
As above, any person, that will use this doc, should be informed, of what is he/she doing in the current step.

Reply:
Divided